portal/src/components/FilterCategories/FilterCategories.component.js

```diff
@@ -128,21 +95,45 @@ export default {
         return category.title_translations[language]
       }
       return category.name
+    },
+    datesRangeFilter() {
+      if (this.selectedFilters) {
+        const datesFilter = this.selectedFilters.find(
+          item => item.external_id === this.publisherDate
+        )
+        if (datesFilter && datesFilter.items) {
+          return {
+            start_date: datesFilter.items[0] || null,
+            end_date: datesFilter.items[1] || null
+          }
+        }
+      } else return {}
     }
   },
   computed: {
-    ...mapGetters(['filter_categories', 'isAuthenticated', 'materials']),
-    /**
-     * generate filtered categories
-     * @returns {*}
-     */
     filtered_categories() {
-      // Return categories which builds the filter tree
-      return this.filter_categories
-        ? _.filter(this.$store.getters.filter_categories.results, {
-            is_hidden: false
-          })
+      // Return categories that build the filter tree
+      let filteredCategories = this.filterCategories
```

Comment:
kunnen we die this.filterCategories niet default `[]` laten zijn? Neemt een hoop condities weg.

Reply:
Lijkt me een goed idee.